guillotina/api/service.py

```diff
@@ -36,6 +40,23 @@ def __setattr__(self, name, value):
 
 
 class Service(View):
+    async def validate(self, parameters):
+        data = await self.request.json()
+        schema = parameters[0]['schema']['$ref'].split('/')[-1]
```

Comment:
Let's validate this in a more safe way. This can quite easily cause errors right now!

Additionally, let's get the schema and make sure it is correct *before* we even get here(when building the service in the `configure/__init__.py` file)